### PR TITLE
Create animal table, insert data into animals table, run quarry.

### PR DESCRIPTION
**Pull request by saikatgpae**
Used [curriculum-template-databases](https://github.com/microverseinc/curriculum-template-databases) to generate new repo.
Created a database named vet_clinic.
Created a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
Copied the SQL used in the previous point into the schema.sql file in the generated repository.
Inserted the following data:
Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
Copied the SQL used in the previous point into the data.sql file in the generated repository.

Wrote quarries as below along with result screenshot.
Find all animals whose name ends in "mon".
Screenshot 2022-11-02 at 11 27 37 PM
List the name of all animals born between 2016 and 2019.
Screenshot 2022-11-02 at 11 31 04 PM
List the name of all animals that are neutered and have less than 3 escape attempts.
Screenshot 2022-11-02 at 11 32 43 PM
List the date of birth of all animals named either "Agumon" or "Pikachu".
Screenshot 2022-11-02 at 11 33 39 PM
List name and escape attempts of animals that weigh more than 10.5kg
Screenshot 2022-11-02 at 11 34 35 PM
Find all animals that are neutered.
Screenshot 2022-11-02 at 11 35 26 PM
Find all animals not named Gabumon.
Screenshot 2022-11-02 at 11 36 27 PM
Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Screenshot 2022-11-02 at 11 37 45 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# curriculum-databases-projects-template
+# Vet clinic database
```
Comment (1GORDON) [inline README.md:1]: Great job @saikatgpae 🤝

- [ ] [Optional] Kindly fix your issues link as it appears broken at the moment.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @saikatgpae ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (saikatgpae) [reply README.md:1]: Hi @1GORDON , 
Thank you for your suggestion.
Updated the issue page link & the same will reflect after the next merge.

**Review by KayLemba — COMMENTED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @saikatgpae        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R







 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**